website/index.html

```diff
@@ -914,8 +917,22 @@
                class="img-fluid pwc-day">
         </a>
       </div>
+        <div class="col-sm-3">
+        <!-- ecroreps -->
+        <a href="https://sustainability.psu.edu/eco-reps" target=_blank>
+            <img src="assets/images/sponsors/ecoreps_original.svg" alt="ecroreps"
+                 class="img-fluid ecoreps_original">
+        </a>
+        </div>
+        <div class="col-sm-3">
+        <!-- EECS -->
+        <a href="https://www.eecs.psu.edu/" target="_blank">
+            <img src="assets/images/sponsors/EECS-day.svg" alt="EECS" class="img-fluid eecs-night"
+            style="padding-top: 17%">
```

Comment:
Is this padding-top due to the weird nature of the EECS logo?

Reply:
I'm pretty positive I just uncommented what was written before for the EECS stuff, so I'm not entirely sure

Reply:
Yeah in that case thats what it is.